src/Common/src/CoreLib/Interop/Windows/Advapi32/Interop.RegistryOptions.cs

```diff
@@ -6,21 +6,21 @@ internal partial class Interop
 {
```

Comment:
This file has more than RegistryOptions. Call it `Interop.Registry.Constants.cs` ?